packages/server/src/index.ts

```diff
@@ -46,8 +48,8 @@ app.post('/auth', async (req, res) => {
 
   res
     .contentType('application/json')
-    .status(200)
     .header('Access-Control-Allow-Origin', '*')
```

Comment:
Do we still need to set the cors header here?

Reply:
We do in production, and this attempts to mirror that.